spec/core/data_structures.md

```diff
@@ -26,10 +26,7 @@ The Tendermint blockchains consists of a short list of data types:
 - [`Validator`](#validator)
 - [`ValidatorSet`](#validatorset)
 - [`Address`](#address)
-- [`ConsensusParams](#consensusparams)
-- [`EvidenceParams`](#evidenceparams)
-- [`ValidatorParams`](#validatorparams)
-- [`VersionParams`](#versionparams)
```

Comment:
why remove these?

Reply:
I considered them under the umbrella of `ConsensusParams`. We can add them back again but in that case we are missing `BlockParams`

Reply:
Yes, block params is missing. I would be in favor of leaving them, technically header, evidence data is under the umbrella of block, but we still have them.

Reply:
Sure, I will update them